package.json

```diff
@@ -1,6 +1,6 @@
 {
   "id" : "netuitive.packages.kubernetes",
-  "version" : "0.2.2",
+  "version" : "0.3.0",
```

Comment:
Typically the version is bumped directly on `dev`, not in feature branches.

Reply:
sounds good, will do that next time.